Cabal-syntax/src/Distribution/Compat/Newtype.hs

```diff
@@ -1,20 +1,21 @@
-{-# LANGUAGE CPP                    #-}
-{-# LANGUAGE DefaultSignatures      #-}
-{-# LANGUAGE FlexibleInstances      #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE FunctionalDependencies #-}
+
 -- | Per Conor McBride, the 'Newtype' typeclass represents the packing and
 -- unpacking of a newtype, and allows you to operate under that newtype with
 -- functions such as 'ala'.
-module Distribution.Compat.Newtype (
-    Newtype (..),
-    ala,
-    alaf,
-    pack',
-    unpack',
-    ) where
+module Distribution.Compat.Newtype
+  ( Newtype (..)
+  , ala
```

Comment:
It looks like the whole codebase was leaning on the side of trailing commas in export lists? If this is configurable, maybe it'd make sense to leave it as a tribute to the past.